src/ImageSharp.Drawing/Processing/TextGraphicsOptions.cs

```diff
@@ -32,7 +32,7 @@ public struct TextGraphicsOptions
 
         private float? dpiY;
 
-        private PixelBlenderMode blenderMode;
+        private PixelColorBlendingMode blenderMode;
```

Comment:
we should add the alpha blending option in here too.

Reply:
@tocsoft I'll take a look